src/conspiracies/pipeline/pipeline.py

```diff
@@ -89,63 +94,47 @@ def docprocessing(self, continue_from_last=False):
         )
 
     def corpusprocessing(self):
-        # TODO: this process is kind of dumb with all the writing and reading of
-        #  files etc., but for now just make it work. It comes from individual scripts
-        #  and a lot of the logic of data structures happen in those read/writes. Also,
-        #  a lot of data that we might want to save is thrown away, e.g. clustered
-        #  entities, or calculated/fetched on the go, e.g. node weights for graphs.
-        docs = (
-            json.loads(line)
-            for line in iter_lines_of_files(
-                f"{self.output_path}/annotations.ndjson",
-            )
-        )
-        n_triplets = 0
-        with open(
-            f"{self.output_path}/triplets.csv",
-            "w+",
-        ) as out:
-            for doc in docs:
-                for triplet in doc["semantic_triplets"]:
-                    triplet_fields = [
-                        triplet["semantic_triplets"][field]["text"]
-                        for field in ("subject", "predicate", "object")
-                    ]
-                    print(", ".join(triplet_fields), file=out)
-                    n_triplets += 1
+        triplets = Triplet.from_annotated_docs(f"{self.output_path}/annotations.ndjson")
```

Comment:
I would generally use pathlib for this:
```suggestion
from pathlib import Path
        output_path = Path(self.output_path) # not needed it it already is a path
        triplets = Triplet.from_annotated_docs( output_path / "annotations.ndjson") 
```

Notably, it handles the fact that "/" is not used on all systems so will cause fewer issues on windows. It also makes specific that something should be a path and not just a string.

Reply:
That looks nice! I'll implement it before merging.